public/standards/index.html

```diff
@@ -1,352 +1,991 @@
 <!DOCTYPE html>
 <html>
```

Comment:
Is the below diff just formatting changes?

Reply:
Good question. I'm not really sure to be honest. I'll rewrite that commit so it is just a one-liner (for now).